services/brig/src/Brig/Data/Client.hs

```diff
@@ -141,7 +141,7 @@ addClientWithReAuthPolicy reAuthPolicy u newId c maxPermClients loc cps = do
   when (reAuthPolicy count upsert) $
     fmapLT ClientReAuthError $
       User.reauthenticate u (newClientPassword c)
-  let capacity = fmap (+ (-count)) limit
+  let capacity = fmap (+ (- count)) limit
```

Comment:
Seems like ormolu dislikes the extra space here.

Reply:
Sorry, I don't understand the review suggestion here.

Reply:
Ah, sorry. I forgot you probably don't have access to our CI logs. The formatter is rejecting the changes because of the extra space after the `-`. 

It wants it back to 

```haskell
let capacity = fmap (+ (-count)) limit
```

Same with the other line that adds a space after `-`.

Reply:
Weird. `make format` explicitly made these changes. Undone now. Thanks!

Reply:
Weird indeed!

Reply:
merge develop and `direnv reload`?  (thanks paolo!)